src/main/java/uk/gov/justice/hmpps/prison/api/resource/PrisonerStatusController.java

```diff
@@ -46,6 +35,7 @@ public class PrisonerStatusController {
             @ApiResponse(responseCode = "200", description = "OK", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = PrisonerInformation.class))}),
             @ApiResponse(responseCode = "401", description = "Unauthorized.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
             @ApiResponse(responseCode = "404", description = "User not found.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})})
+    @VerifyOffenderAccess(overrideRoles = {"SYSTEM_USER", "GLOBAL_SEARCH", "VIEW_PRISONER_DATA"})
```

Comment:
Should we be adding SYSTEM_USER here? I thought it was deprecated...

Reply:
We should only be adding it if we're moving it (as in this case).  The only alternative would be to remove the role if the endpoint has been checked and all associated calling clients don't have that role

Reply:
Oh sorry it's move from the service. I see, thanks 😄

Reply:
Im just leaving it as it is (or was , on the service layer)